web/src/lib/components/AuctionCard.svelte

```diff
@@ -29,7 +29,7 @@
             onView(auction);
         }
 
-        window.open(getUrl(), "_blank");
```

Comment:
Why?

Reply:
I changed it to:
```window.open(getUrl(), "_self");```

so it will open in the same tab